ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -88,6 +90,10 @@ class ShyHeaderController: UIViewController {
         defer {
             contentScrollView = contentViewController.navigationItem.contentScrollView
         }
+
+        accessoryViewObservation = contentViewController.navigationItem.observe(\UINavigationItem.accessoryView) { [unowned self] item, _ in
```

Comment:
Curious - why "unowned" vs. "weak"?

Reply:
i was just following the other pattern in this file. but my guess is that self doesn't have to be optional and it will always have value and not have check if it nil ....